can/simulator/rtos_main.cpp

```diff
@@ -0,0 +1,94 @@
+#include <iostream>
+#include <variant>
```

Comment:
This "simulator" is here to illustrate interacting with the CAN message buffer/dispatching.